src/charm.pyi

```diff
@@ -22,6 +22,7 @@ class WordpressCharm(ops.charm.CharmBase):
     _SERVICE_NAME: str
     _WORDPRESS_USER: str
     _WORDPRESS_GROUP: str
+    _DB_CHECK_INTERVAL: Union[float, int]
```

Comment:
Depending on which version of Python is in use (I think this is only supported Python 3.10 onwards) this could be changed to `float | tuple` I think